repos/roslyn.proj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
 
   <PropertyGroup>
-    <NuGetPackageVersion>2.3.2-beta1-61921-05</NuGetPackageVersion>
+    <NuGetPackageVersion>2.10.0-beta2-63410-10</NuGetPackageVersion>
```

Comment:
I don't think this has been affecting roslyn package version (anymore?). A build I did before this PR produced e.g. `Microsoft.CodeAnalysis.CSharp.2.8.0` even though this was `2.3.2-beta1-61921-05`. Will comment here again with the version this PR ends up producing.

Doesn't seem like a problem because of forced auto-update, but we could get rid of this to be less misleading.

Reply:
Yeah, produces `Microsoft.CodeAnalysis.CSharp.2.10.0` now.